support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouPasswordSet.jsx

```diff
@@ -18,6 +21,11 @@ type PropTypes = {
 
 // ----- Render ----- //
 function ContributionThankYouPasswordSet(props: PropTypes) {
+  const store = pageInit(() => initReducer(), true);
+  const state = store.getState();
+  const { countryId } = state.common.internationalisation;
```

Comment:
`countryId` can be passed in as a prop by the parent

Reply:
✅